tests/transformers/test_transformers_model_export.py

```diff
@@ -3787,3 +3947,69 @@ def predict(self, context, model_input, params=None):
                 python_model=TestModel(),
                 artifacts={"some-model": "hf:/invalid-repo-id"},
             )
+
+
+def test_model_distributed_across_devices():
```

Comment:
instead of merging master into your branch, could you rebase? These test additions are from another PR.

Reply:
Yes added https://github.com/mlflow/mlflow/pull/10041